src/main/java/org/jabref/gui/groups/GroupDialog.java

```diff
@@ -610,7 +616,21 @@ private void openBrowseDialog() {
             .addExtensionFilter(StandardFileType.AUX)
             .withDefaultExtension(StandardFileType.AUX)
             .withInitialDirectory(Globals.prefs.get(JabRefPreferences.WORKING_DIRECTORY)).build();
-        dialogService.showFileOpenDialog(fileDialogConfiguration).ifPresent(file -> texGroupFilePath.setText(file.toAbsolutePath().toString()));
+        dialogService.showFileOpenDialog(fileDialogConfiguration).ifPresent(file -> texGroupFilePath.setText(relativize(file.toAbsolutePath()).toString()));
+    }
+
+    private Path relativize(Path path) {
+        List<Path> fileDirectories = getFileDirectoriesAsPaths();
+        return FileUtil.relativize(path, fileDirectories);
+    }
+
+    private List<Path> getFileDirectoriesAsPaths() {
+        List<Path> fileDirs = new ArrayList<>();
+        MetaData metaData = basePanel.getBibDatabaseContext().getMetaData();
+        metaData.getLaTexFileDirectory(prefs.getFilePreferences().getUser())
+                .ifPresent(LaTexFileDirectory -> fileDirs.add(Paths.get(LaTexFileDirectory).toAbsolutePath().normalize()));
```

Comment:
The metadata should already return the directory as a path object. Using strings often leads to duplicate code (as happened here too, this exact line can be found further below too) and one should thus try to use special objects as often as possible. 

In the same spirit, I would prefer if the setter in `metadata` also accepts a path.